src/QAtomicIntCompat.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_QATOMICINTCOMPAT_H_
+#define MUMBLE_QATOMICINTCOMPAT_H_
+
+// Portably (between Qt 4 and Qt 5) load the value
+// of a QAtomicInt.
+inline int QAtomicIntLoad(const QAtomicInt &ai) {
```

Comment:
Any specific reason to uppercase the name? Makes me think it's a class or something at point of use.